Simple modification to make it compatible with Fisher

ngmoviedo opened the pull request:
I know that this plugin is made for Oh-my-fish, but it also works with Fisher, with only one caveat: the key bindings don't get imported automatically. This can be easily solved by moving the key_bindings.fish file to the conf.d directory and, preferably, giving to the file a less generic name to avoid conflicts with other plugins.

ngmoviedo commented:
I'm not completely sure if this second commit is compatible with omf.

jorgebucaran commented:
@ngmoviedo It is. 👍

jorgebucaran commented on functions/README_budspencer.md:
```diff
@@ -0,0 +1,291 @@
+# Budspencer theme
```
@ngmoviedo What's the purpose of this?

jorgebucaran replied:
I am not a fan of further cluttering my `functions` directory. For Fisher, I'd suggest removing this feature completely.

ngmoviedo commented:
I moved the readme file to the functions directory and added a symlink to it in the root directory. The name has been changed to `README_budspencer.md` to make it less generic and avoid possible conflicts. That way it also gets downloaded by Fisher. The function `budspencer_help` (which displays the readme file) in the `fish_prompt.fish` script has been modified to adapt to this change. Now it should be able to display the help file for both omf and fish users.

jorgebucaran reviewed (approved):
LGTM

jorgebucaran commented on functions/fish_prompt.fish:
```diff
@@ -89,9 +89,9 @@ end
+  if test -f $readme_file_omf
     set -l readme_file "$readme_file_omf"
-  elif [ -e $readme_file_fisher ]
+  else if test -f $readme_file_fisher
```
@ngmoviedo Thank you, `test` is much more readable. 👍

scorphus reviewed (commented):
I'd close this in favour of #79 as that is cleaner and changes only the necessary files.

ngmoviedo commented:
I know I changed more that it was needed, so probably you're right. Lately I was maintaining it mostly for personal use.

ngmoviedo commented:
Anyway, moving the README.md also fixes a compatibility issue with fisher. And the modification in `fish_greeting.fish` were added to fix the issue I was experiencing, mentioned in #75. It solves the missing greeting, at least for me.
